README.md

```diff
@@ -39,6 +39,17 @@ To setup on your local machine:
 
 **NOTE** - The [Alternating Least Squares (ALS)](notebooks/00_quick_start/als_movielens.ipynb) notebooks require a PySpark environment to run. Please follow the steps in the [setup guide](SETUP.md#dependencies-setup) to run these notebooks in a PySpark environment.
 
+## Install this repository via PIP
+A [setup.py](reco_utils/setup.py) file is provied in order to simplify the installation of this utilities in this repo from the main directory. 
+
+    pip install -e reco_utils
+
+It is also possible to install directly from Github.
+
+    pip install git+http://github.com/microsoft/recommenders/#egg=pkg&subdirectory=reco_utils
```

Comment:
It failed installing from GitHub directly.  I am not sure whether it is the problem of pip to install from pull request.  I tried reco_utils:
```console
$ pip install -e git://github.com/microsoft/recommenders.git@refs/pull/851/head#egg=reco_utils\&subdirectory=reco_utils
Obtaining reco_utils from git+git://github.com/microsoft/recommenders.git@refs/pull/851/head#egg=reco_utils&subdirectory=reco_utils
  Cloning git://github.com/microsoft/recommenders.git (to revision refs/pull/851/head) to ./src/reco-utils
  Running command git clone -q git://github.com/microsoft/recommenders.git /home/test/Test/src/reco-utils
  WARNING: Did not find branch or tag 'refs/pull/851/head', assuming revision or ref.
  Running command git fetch -q git://github.com/microsoft/recommenders.git refs/pull/851/head
  Running command git checkout -q ec1ce832b3aaaa2f8889b92d33a64d75981527e6
Installing collected packages: reco-utils
  Found existing installation: reco-utils 2019.6
    Uninstalling reco-utils-2019.6:
      Successfully uninstalled reco-utils-2019.6
  Running setup.py develop for reco-utils
Successfully installed reco-utils
$ ls
src
$ ls src/
pip-delete-this-directory.txt  reco-utils
$ python
Python 3.7.3 (default, Mar 27 2019, 22:11:17) 
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import reco_utils
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'reco_utils'
>>> 
```
and utils_nlp:
```console
$ pip install -e  git+git@simonzhaomsgithub:microsoft/nlp#egg=utils_nlp
Obtaining utils_nlp from git+git@simonzhaomsgithub:microsoft/nlp#egg=utils_nlp
  Cloning git@simonzhaomsgithub:microsoft/nlp to ./src/utils-nlp
  Running command git clone -q git@simonzhaomsgithub:microsoft/nlp /home/simon/Copy/Test/cvbp/src/utils-nlp
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... done
Installing collected packages: utils-nlp
  Running setup.py develop for utils-nlp
Successfully installed utils-nlp
$ ls
src
$ ls src/
pip-delete-this-directory.txt  utils-nlp
$ python
Python 3.7.3 (default, Mar 27 2019, 22:11:17) 
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import utils_nlp
>>>
```

Reply:
Can you try using the branch instead of the PR ref? @gramhagen/pip#egg...

Reply:
@simonzhaoms I updated the instructions in the readme this is the format you need:

```
pip install -e git+https://github.com/microsoft/recommenders/@branch#egg=pkg\&subdirectory=reco_utils
```